javalib/src/main/scala/java/util/AbstractSequentialList.scala

```diff
@@ -0,0 +1,42 @@
+package java.util
+
+abstract class AbstractSequentialList[E] protected ()
+    extends AbstractList[E] {
+
+  def get(index: Int): E =
+    listIterator(index).next
```

Comment:
The `listIterator(index)` would throw the `IndexOutOfBounds` before the `.next()` is called. Therefore the correct exception will be thrown.